sopel/modules/url.py

```diff
@@ -231,6 +259,18 @@ def get_hostname(url):
     return hostname
 
 
+def get_tinyurl(url):
+    """ Returns a shortened tinyURL link of the URL. """
+    tinyurl = "https://tinyurl.com/api-create.php?url=%s" % url
+    try:
+        res = requests.get(tinyurl)
```

Comment:
For additional robustness, raise an exception for HTTP error status codes as well:

```suggestion
        res = requests.get(tinyurl)
        res.raise_for_status()
```

This will guard against changes in the TinyURL API that result in HTTP 4xx errors, or API downtime that is severe enough to return HTTP 5xx errors but not so severe that the API cannot accept and process connections.